service/src/main/java/bio/terra/pipelines/app/controller/PipelinesApiController.java

```diff
@@ -47,8 +42,8 @@ public PipelinesApiController(
 
   private static final Logger logger = LoggerFactory.getLogger(PipelinesApiController.class);
 
-  private SamUser getAuthenticatedInfo() {
-    return samUserFactory.from(request, samConfiguration.baseUri());
+  private void getAuthenticatedInfo() {
```

Comment:
is this change because we don't actually use the `SamUser` output of this method? should we rename to something like `checkUserInSam` ?